lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ipaddress
+import grpc
+import logging
+
+from magma.subscriberdb.sid import SIDUtils
+
+
+class SubscriberDbClient:
+    def __init__(self, rpc_stub):
+        self.subscriber_client = rpc_stub
+
+    def get_subscriber_ip(self, sid: str) -> ipaddress.ip_address:
+        """
+        Make RPC call to 'GetSubscriberData' method of local SubscriberDB
+        service to get assigned IP address if any.
+        """
+        if self.subscriber_client is None:
+            return None
+
+        ip_addr = None
+        try:
+            apn_name = ''
+
+            if '.' in sid:
+                apn_name = sid.split('.')[1]
```

Comment:
`imsi, apn_name = sid.split('.')` to unpack the values from the splitted list since we know the expected num of values returned